nucypher/policy/conditions/lingo.py

```diff
@@ -44,7 +45,14 @@ def to_json(self) -> str:
 
 
 class ReturnValueTest:
-    COMPARATORS = ('==', '>', '<', '<=', '>=')
+    _COMPARATOR_FUNCTIONS = {
+        '==': lambda x, y: x == y,
```

Comment:
@cygnusv side note: found these python builtin methods you can use instead of the lambdas,https://docs.python.org/3/library/operator.html;  up to you if you want to use them or not.